lib/bundle/core+encrypt+promise.js

```diff
@@ -5,8 +5,12 @@ import Promise from '../promise';
  /**
   * @param {object} [cfg]
   */
-export default function Ravelin(cfg) {
-  cfg.Promise = cfg.Promise || window.Promise || Promise;
+function Ravelin(cfg) {
+  cfg.Promise = cfg.Promise || Ravelin.Promise || window.Promise || Promise;
```

Comment:
I think the `Ravelin.Promise = ...` assignment below means the last two vars won't ever be reached, based on the usage in the non-`+promise` bundles.

Reply:
Woops, looks like I made a hash of these cfg.Promise changes. Updated them all in https://github.com/unravelin/ravelinjs/pull/42/commits/e33a86b69ee98f043150a2607f053fec1b563a5c